presto-docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -7,6 +7,8 @@ external `Microsoft SQL Server <https://www.microsoft.com/sql-server/>`_ databas
 different systems like SQL Server and Hive, or between two different
 SQL Server instances.
 
+SQL Server 2017+ is supported. This connector is tested against 2019-CU6.
```

Comment:
> This connector is tested against 2019-CU6.
 

This is imprecise when some tests are run against 2017 and some agsinst 2019.

Also, it either is meaningless to end users (they don't care about how we test and whether we test)
or it can be taken as an attempt to blur "is supported" sentence.

Let's remove it